app/controllers/pages_controller.rb

```
@@ -2,7 +2,8 @@
 
 class PagesController < ApplicationController
   include ReactOnRails::Controller
-  before_action :set_comments
+  include ReactOnRailsPro::Stream
```

RuboCop (1.86.1)

[convention] Add an empty line after module inclusion.

(Layout/EmptyLinesAfterModuleInclusion)